test/neon_animated_pages_test.dart

```diff
@@ -0,0 +1,45 @@
+@TestOn('browser')
+library polymer_elements.neon_animated_pages_test;
+
+import 'package:polymer_interop/polymer_interop.dart';
+import 'package:polymer_elements/neon_animated_pages.dart';
+import 'package:test/test.dart';
+import 'package:web_components/web_components.dart';
+import 'common.dart';
+import 'dart:html';
+
+main() async {
+
+  await initWebComponents();
+
+  group('<neon-animated-pages>', () {
+
+    group('basic',(){});
+
+    group('notify-resize',(){
+
+      test('only a destination page receives a resize event',() async {
+        NeonAnimatedPages animatedPages = fixture('notify-resize');
+        List<Element> resizables = Polymer.dom(animatedPages).children;
+        Map receives = {};
+        resizables.forEach((Element page){
+          page.addEventListener('iron-resize',(Event event){
```

Comment:
I have been changing these over to the `on` syntax (`page.on['iron-resize'].listen((Event event) {`), but its not entirely necessary. Mostly I do that so I can use `.take('1')` to automatically cancel the listener, but in this case that is less useful since you actually want to keep listening to make sure you don't get more calls than usual.